examples/stable/test/property/ic_api/in_replicated_execution/test/tests.ts

```diff
@@ -70,7 +70,7 @@ export function getTests(): Test {
             ).toStrictEqual([true]);
         });
 
-        it('redeploys the canister', async () => {
+        please('redeploys the canister', async () => {
```

Comment:
Should read `please redeploy the canister`

Reply:
Done